### DISCUSSION TITLE
auto-tags support for includeProcessGroupReferences when downloading

### DISCUSSION THREAD

**Discussion by kristofre** [Dynatrace/dynatrace-configuration-as-code#407] [Ideas]
Currently, when downloading `auto-tags,` it does not download the ProcessGroup References as they are only retrieved when setting the parameter `includeProcessGroupReferences` is set to `true`.

This means that there is a discrepancy between the UI and the downloaded configuration.
Can we support this parameter?

Thanks!

Kristof

**Comment by tatomir146** [comment]
Process group references aren't compatible across environments. We wouldn't be able to upload this configuration to another env then.
